### DISCUSSION TITLE
How to set up a hydrogen combustion reaction

### DISCUSSION THREAD

**Discussion by gdut21122** [firemodels/fds#12677] [General]
In reactions, select Add From Library, and select HYDROGEN. In Fuel Species, select HYDROGEN or REAC-FUEL.At the same time, why the hydrogen combustion reaction temperature I set is very low, only tens of degrees, is the reason not to produce smoke?

**Comment by Er9y714** [comment]
1. I guess you refer to Pyrosim interface. 
2. What value do you refer in terms of FDS parameter?
3. Do you mean soot by smoke?

**Comment by drjfloyd** [comment]
Make sure you have read all of the FDS (and not just Pyrosim) documentation. This is the first source of information for questions on FDS.
